CHARTER.md

```diff
@@ -1,25 +1,16 @@
-# Open Container Initiative Charter v1.1
+# Open Container Initiative Charter v1.2
 
 | Comment          | Effective Date   |
 | ---------------- |:----------------:|
 | Initial release  | 13 November 2015 |
-| Update           | 6 May 2020       |
+| Update           | 18 May 2020       |
```

Comment:
I think this table is meant to be appended to on each update. Actually the `Update` entry from the first update last week probably should've been

```suggestion
| v1.1 (update OCI scope)           | 6 May 2020        |
| v1.2 (further clarify OCI scope)  | 18 May 2020       |
```